### DISCUSSION TITLE
支持 从mysql 单向实时同步到 stonedb不

### DISCUSSION THREAD

**Discussion by flyowl** [stoneatom/stonedb-docs#134] [Q&A]
想把业务变成读写分离，stonedb用来读，应该效果不错

**Comment by fmpoffice** [comment]
读写都用stoneDb不是更好吗？如果光读还不如直接上Elasticsearch，也支持sql语句的

**Comment by Nliver** [comment]
请您到主仓库提交issue：https://github.com/stoneatom/stonedb/issues
Please post this issue to our main repo.
